src/server/misc/Utilities.ts

```diff
@@ -5,10 +5,19 @@ import * as vm from 'vm';
 import { ICompilerFile } from '../compiler';
 
 enum AllowedInternalModules {
+    assert,
+    async_hooks,
+    buffer,
+    crypto,
+    dns,
```

Comment:
This is the only one I'm unsure about, since it would allow apps to make a call outside of our app apis 🤔